java/client/src/main/java/glide/api/commands/HyperLogLogBaseCommands.java

```diff
@@ -0,0 +1,51 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "HyperLogLog Commands" group for standalone clients
+ * and cluster clients.
+ *
+ * @see <a href="https://redis.io/commands/?group=hyperloglog">HyperLogLog Commands</a>
+ */
+public interface HyperLogLogBaseCommands {
+
+    /**
+     * Adds all the elements to the HyperLogLog data structure stored and create a new structure if it
+     * is missing.
+     *
+     * <p>As a side effect of this command the HyperLogLog internals may be updated to reflect a
```

Comment:
Side effects are too much detail.  We should just have the high-level details of the command's effects.  See other commands for an example.  Something like: 
<img width="627" alt="image" src="https://github.com/Bit-Quill/glide-for-redis/assets/104530826/5f9e4d60-d8c4-46d1-94d4-5fbbe6105e02">

Reply:
In some cases, they use the first paragraph of redis.io documentation (example: persist), and in other cases they include a custom description (pttl, for example, seems customized).